packages/cxl-lumo-styles/scss/themes/dominik-vaadin-accorion.scss

```diff
@@ -0,0 +1,33 @@
+:host([theme~="cxl-dashboard-sidebar"]) {
```

Comment:
What's up with this filename `dominik-vaadin-accorion.scss`

Reply:
this was only backup file made before the Greg's branch merge, deleted